src/coordinator/users.js

```diff
@@ -510,6 +519,86 @@ class Users {
       `${this._baseUrl}/users/${userId}/stacks/${stackId}`
     );
   }
+
+  /**
+   * Subscribes a user to an event
+   *
+   * API Endpoint: '/users/:userId/events/:event_id'
+   * Method: POST
+   *
+   * @param {string} userId The ID of the user
+   * @param {string} eventId The ID of the event
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtUserEvent} The newly created user event
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator.users
+   *   .subscribeEvent('36b8421a-cc4a-4204-b839-1397374fb16b', '007ca9ee-ece7-4931-9d11-9b4fd97d4d58')
```

Comment:
For the example userId, can you make it more like what the userIds look like -- they are more like `auth0|sakl;fjheuaiweh` or `okta|jfkasldofhea` than an UUID

Reply:
I can do that. That makes more sense